src/main/java/nl/tudelft/dnainator/ui/drawables/phylogeny/LeafNode.java

```diff
@@ -56,26 +52,14 @@ public void onMouseClicked() {
 	}
 
 	@Override
-	public void setInactive(boolean state) {
-		if (highlighted) {
-			removeStyles();
-			ColorServer.getInstance().revokeColor(node.getName());
-			highlighted = false;
-		}
-		inactive.set(state);
-	}
-
-	@Override
 	protected void addStyle(String style) {
 		shape.getStyleClass().add(style);
-		incomingEdge.getStyleClass().add(style + "-edge");
```

Comment:
For this PR it has been removed, because of the switch to storing outgoing edges. I didn't implement this yet because I felt it was more related to the subtree highlighting task.